### PR TITLE
Updated the description on how to update a delegation during a signing

### PR DESCRIPTION
**Pull request by kommendorkapten**
Adds a section for the orchestrator on how to update a delegation during a signing ceremony. 
[Rendered version](https://github.com/kommendorkapten/root-signing/blob/update-delegation-steps/playbooks/ORCHESTRATION.md#step-4-update-delegations-optional)

#### Release Note
N/A

#### Documentation
This PR only updates the documentation.

### CODE REVIEW

File: playbooks/ORCHESTRATION.md [hunk 1]
```diff
@@ -229,12 +318,12 @@ This will create a PR signing the snapshot and timestamp files and committed the
...
-./scripts/verify.sh $PR
+$ GITHUB_USER=<your GitHub handle> ./scripts/verify.sh $PR
 ```
 
 Note: You cannot test this step locally against the current staged repository, since the snapshot and timestamp keys are only given permissions to the GitHub Workflows. However, under the hood, the workflow is running `./scripts/step-3.sh` and `./scripts/step-4.sh`. If you initialize a ceremony with local testing keys, this action will work.
 
-## Step 6: Publication
+## Step 7: Publication
 
 Once the PR from [Step 3](#step-3-snapshotting-and-timestamping) is merged, a [workflow](../.github/workflows/sync-ceremony-to-main.yml) will automatically create a PR merging the changes on the completed ceremony branch to main.
```
Comment (Hayden-IO) [inline playbooks/ORCHESTRATION.md:357]: Can we remove the extra step under Other? https://github.com/sigstore/root-signing/blob/e728003cce59f5e15c4d2a546e5edad6b76ceff5/playbooks/ORCHESTRATION.md#adding-a-delegation

File: playbooks/ORCHESTRATION.md [hunk 2]
```diff
@@ -193,7 +193,96 @@ $PR` runs on the PR to validate the `targets.json` has a valid
...
+Now add back the previous delegation file and update it:
+
+```shell
+$ cp repository/${DELEGATION_NAME}.json repository/staged
+$ cp -r /path/to/delegated/targets .
+$ cp /path/to/delegate-meta.yaml .
+$ tuf add-delegation \
+      -repository repository \
+      -name ${DELEGATION_NAME} \
+      -target-meta delegate-meta.yaml \
```
Comment (Hayden-IO) [inline playbooks/ORCHESTRATION.md:248]: Can we provide an example of a delegate-meta.yaml file, or check one into root-signing?

**Review by Hayden-IO — DISMISSED** [review]
Looks great! Just two small comments.

File: playbooks/ORCHESTRATION.md (hunk 2 above)
Comment (kommendorkapten) [reply playbooks/ORCHESTRATION.md:248]: Fixed!

**Review by Hayden-IO — APPROVED** [review]
Looks great!

**Pull request merged**